.gitignore

```diff
@@ -0,0 +1,53 @@
+# These are some examples of commonly ignored file patterns.
```

Comment:
I like having a standard .gitignore that we reuse across projects.

Reply:
@khrisnakamarga do you know which .gitignore I should use?